src/main/java/com/joejoe2/chat/repository/channel/PrivateChannelRepository.java

```diff
@@ -39,6 +39,12 @@ default Slice<PrivateChannel> findByIsUserInMembers(User user, Pageable pageable
     return findByMembersContainingUserByUpdateAtDesc(user, pageable);
   }
 
+  @Query(
+      "SELECT DISTINCT ch from User u "
+          + "join u.blockedPrivateChannels ch where u = :user "
+          + "ORDER BY ch.createAt DESC")
+  Slice<PrivateChannel> findBlockedByUser(User user, Pageable pageable);
```

Comment:
needsta be private

Reply:
private ? The method is in a interface and used by service.

Reply:
Apologies, I didn't notice that that's an interface, I thought I placed a comment into the JPA entity. :)